Changes

```diff
@@ -1,4 +1,4 @@
-0.55.2.x (relative to 0.55.1.0)
+0.55.2.0 (relative to 0.55.1.0)
```

Comment:
I thought the plan with these was to leave them as `x.x` until release?

Reply:
I'd accidentally put the `x` in during the merge forward, as thats what I as thinking.   But then I realised that it has to be `55.2` as it has new features. I think if anything, I'd rather we just left off the header at the top, as it also makes the conflict resolution easier to mess up. So putting it back to `0` just makes it as it was in that branch... I think you're right that in practice this does get a bit confusing.